banman/store.go

```diff
@@ -0,0 +1,219 @@
+package banman
+
+import (
+	"encoding/binary"
+	"errors"
+	"net"
+	"time"
+
+	"github.com/btcsuite/btcwallet/walletdb"
+)
+
+var (
+	// byteOrder is the preferred byte order in which we should write things
+	// to disk.
+	byteOrder = binary.BigEndian
```

Comment:
Wondering if this is always needed when persisting numerical values to disk?

Reply:
We're not just persisting numerical values in the store. Not sure I follow your question.